MCUMgr Sample Fails to build

**Describe the bug**
MCUMgr sample is failing to build on mimxrt1060_evk, although it looks like the issue would be present on any board.

**To Reproduce**
Steps to reproduce the behavior:
1. mkdir build; cd build
2. cmake -DBOARD=mimxrt1060_evk samples/subsys/mgmt/mcumgr/smp_svr
3. make
4. See error

**Expected behavior**
Build should pass

**Logs and console output**
```
/home/danieldegrasse/zephyrproject/zephyr/subsys/mgmt/mcumgr/lib/cmd/os_mgmt/src/zephyr_os_mgmt.c:8:10: fatal error: power/reboot.h: No such file or directory
    8 | #include <power/reboot.h>
      |          ^~~~~~~~~~~~~~~~
```

--- a/subsys/mgmt/mcumgr/lib/cmd/os_mgmt/src/zephyr_os_mgmt.c
+++ b/subsys/mgmt/mcumgr/lib/cmd/os_mgmt/src/zephyr_os_mgmt.c
@@ -5,7 +5,7 @@
  */
 
 #include <zephyr.h>
-#include <power/reboot.h>
+#include <sys/reboot.h>
 #include <debug/object_tracing.h>
 #include <kernel_structs.h>
 #include <mgmt/mgmt.h>

